app/products/calls/components/calls_badge.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import {typography} from '@utils/typography';
+
+const styles = StyleSheet.create({
+    container: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'center',
+        borderRadius: 4,
+    },
+    recording: {
+        paddingLeft: 8,
+        paddingRight: 8,
+        backgroundColor: '#D24B4E',
+        color: 'white',
```

Comment:
Hardcoding colors here and elsewhere because we're not theming the call screen (yet).